File: GigRadio/Controllers/MusicEngine.m
//
//  MusicEngine.m
//  GigRadio
//
//  Created by Michael Forrest on 25/10/2014.
//  Copyright (c) 2014 Good To Hear. All rights reserved.
//

#import "MusicEngine.h"

@implementation MusicEngine
-(void)playSoundCloudTrack:(SoundCloudTrack *)track{
    [self playUrl:track.playbackURL];
    self.currentlyPlayingTrack = track;
}
@end

File: GigRadio/Controllers/MusicEngine.swift
//
//  MusicEngine.swift
//  GigRadio
//
//  Created by Michael Forrest on 17/05/2015.
//  Copyright (c) 2015 Good To Hear. All rights reserved.
//

import UIKit
class MusicEngine {
    var currentTrack: SoundCloudTrack?
    func playSoundCloudTrack(track: SoundCloudTrack){
//        playUrl(track.playbackURL)
        currentTrack = track
    }
}
